test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -94,7 +94,7 @@ func TestAPIExportVirtualWorkspace(t *testing.T) {
 	framework.AdmitWorkspaceAccess(t, ctx, kubeClusterClient, serviceProviderWorkspace, []string{"user-1", "user-2"}, nil, []string{"access"})
 	framework.AdmitWorkspaceAccess(t, ctx, kubeClusterClient, consumerWorkspace, []string{"user-3"}, nil, []string{"admin"})
 
-	cr, crb := createClusterRoleAndBindings("user-3-binding", "user-3", "User", "apiexports", "today-cowboys", []string{"bind"})
```

Comment:
for anyone keeping track, it's the loss of velocity & extra burden caused by generic test helpers which I don't like ;)

Reply:
sorry, i don't follow :-) do you mind to elaborate? Generally I am not too happy about my factoring of `createClusterRoleAndBindings` either. Putting those struct initialziations here though makes the code size huge. I am fine doing this though if it seems more idiomatic and leave `createClusterRoleAndBindings` as is, wdyt?

Reply:
We've had a lot of conversations in the team over the last N months about mega-helpers in tests, and I've always said to keep things simple & isolated, or you end up with this type of refactor burden on YOU in this PR, which is not ideal. You should do whatever you think is simpler and quicker for you here.